src/hoc/selectable-enhance.js

```diff
@@ -76,12 +76,20 @@ export const SelectableContainerEnhance = (Component) => {
           key: this._keyIndex,
           style: mergedChildrenStyles,
           nestedItems: child.props.nestedItems.map((child) => this._extendChild(child, styles, selectedItemStyle)),
+          initiallyOpen: child.props.nestedItems.reduce(this._hasSelectedDescendant, false),
         });
       } else {
         return child;
       }
     },
 
+    _hasSelectedDescendant(previousValue, child) {
+      if (child.props.nestedItems.length > 0) {
```

Comment:
if a child doesn't have `nestedItems` or if the child is not a valid react element this will result in an error. 
```js
if (React.isValidElement(child) && child.props.nestedItems && child.props.nestedItems.length > 0) {
  // ...
}

```